lib/union_of.rb

```diff
@@ -0,0 +1,299 @@
+# frozen_string_literal: true
+
+require 'active_record_union'
```

Comment:
Is this dependency needed? IIRC, we're using Arel everywhere we were previously using this gem.